geomstats/geometry/hypersphere.py

```diff
@@ -303,6 +303,40 @@ def random_von_mises_fisher(self, kappa=10, n_samples=1):
 
         return point
 
+    def random_von_mises_fisher2(self, kappa=10, n_samples=1):
```

Comment:
Is there a difference between this method and the one just above (`random_von_mises_fisher`) ?

Reply:
oh sorry was doing some tests and forgot to remove it. thanks